pkg/webui/console/store/reducers/users.js

```diff
@@ -21,11 +21,14 @@ import {
   GET_USER,
   DELETE_USER_SUCCESS,
   CREATE_USER_SUCCESS,
+  GET_USER_INVITATIONS_SUCCESS,
 } from '@console/store/actions/users'
 
 const initialState = {
   entities: {},
   selectedUser: null,
+  invitations: undefined,
```

Comment:
```suggestion
  invitations: [],
```
This will save you headaches when defining selectors (due to having to do type checks). Also note that (iirc) redux will strip any values that are `undefined`, meaning that setting a value to `undefined` is the same as not setting a value at all.